index.bs

```diff
@@ -119,6 +119,19 @@ To <dfn>Construct an AmbientLightSensor Object</dfn> the user agent must invoke
 The <a attribute for="AmbientLightSensor">illuminance</a> attribute of the {{AmbientLightSensor}}
 interface represents the <a>current light level</a>.
 
+Use Cases and Requirements {#usecases-requirements}
+=========
+
+1. A Web application gradually updates document style based on light level changes.
```

Comment:
Quite a few [references](https://www.w3.org/Search/Mail/Public/search?keywords=light-level&hdr-1-name=subject&hdr-1-query=&index-grp=Public_FULL&index-type=t&type-index=www-style) in the mailing list. Seems there are accessibility-related concerns here which seems is why the feature is on hold within the CSS WG.